pkgs/applications/system/mgmt/default.nix

```diff
@@ -0,0 +1,65 @@
+{ augeas
+, buildGoModule
+, fetchFromGitHub
+, gotools
+, lib
+, libvirt
+, libxml2
+, nex
+, pkg-config
+, ragel
+, runtimeShell
+}:
+let
+  rev = "d8820fa1855668d9e0f7a7829d9dd0d122b2c5a9";
+in
+buildGoModule rec {
+  pname = "mgmt";
+  version = "0.0.22-20221024063838-${builtins.substring 0 12 rev}";
```

Comment:
Please use the unstable-XXXX-XX-XX format where XXXX-XX-XX is the date of the commit. The git hash is not required here.